client/components/repository/Outline/SearchResult.vue

```diff
@@ -54,4 +59,9 @@ export default {
     display: none;
   }
 }
+
+.v-sheet {
```

Comment:
The above v-sheet is wrongly used (it is using card components within). Once you switch it to v-card you can make it so it is selectable and you can remove cursor: pointer. Slip on my end, sorry.